project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.3.13
+sbt.version = 1.2.8
```

Comment:
why? i mean, it was necessary?

Reply:
Me and @sevanspowell were pulling our hair on this all night... briefly, given that:
- Sbtix doesn't work correctly with SBT 1.3 (it creates a `repo.nix` that references local path)
- We don't really know how to fix Sbtix for SBT 1.3
- We don't know how to estimate how long it would take to figure it out
- We have some aggressive time constraints
- SBT 1.3 is at best a nice-to-have (this is an assumption, correct me if I'm wrong)
- Other urgent tasks are blocked by this not being merged

therefore we made an executive decision to go back to `1.2.8` which was already known to work...

Reply:
first of all thanks for the clarification and all the effort you are doing! Yeah, fully agree, it was not a requirement. We can get rid off it, given are current deadlines. (On devops-sync call we also agreed upon that)